app/views/nonprofit_interest_forms/new.html.slim

```diff
@@ -41,9 +42,10 @@
     = f.input :category, collection: NonprofitInterestForm::CATEGORIES, as: :radio_buttons
     = f.input :org_description
     = f.input :referrer
-    = f.input :submitted_calendly, label: 'Please sign up for a phone call with us using <a  target="_blank" href="https://calendly.com/calblueprint/npo-learn-more">this link</a>'.html_safe
-
-
+    <link href="https://assets.calendly.com/assets/external/widget.css" rel="stylesheet">
+    <script src="https://assets.calendly.com/assets/external/widget.js" type="text/javascript"></script>
+    = f.input :submitted_calendly, label: 'Please sign up for a phone call with us <a href="" onclick="Calendly.initPopupWidget({url: \'https://calendly.com/calblueprint/npo-learn-more\'});return false;">here</a>'.html_safe
```

Comment:
Change to: Please sign up for a phone call with us [through this Calendly link](URL)

Also, not a huge deal but maybe add for the little detail line something that says it's **mandatory**